nyaa/templates/rss.xml

```diff
@@ -40,6 +40,11 @@
 			<nyaa:categoryId>{{- cat_id }}</nyaa:categoryId>
 			<nyaa:category>  {{- category_name(cat_id) }}</nyaa:category>
 			<nyaa:size>      {{- torrent.filesize | filesizeformat(True) }}</nyaa:size>
+			<nyaa:submitter> {{- 'Anonymous' if torrent.anonymous or not torrent.user else torrent.user.username }}</nyaa:submitter>
+			<nyaa:trusted>	 {{- torrent.trusted }}</nyaa:trusted>
+			<nyaa:remake>	 {{- torrent.remake }}</nyaa:remake>
```

Comment:
These should rather be explicit strings, and instead of `True` and `False` I'd prefer `Yes` and `No` (`torrent.remake and 'Yes' or 'No'`).

Reply:
Fixed.